src/bun.js/javascript.zig

```diff
@@ -2735,7 +2735,7 @@ pub fn NewHotReloader(comptime Ctx: type, comptime EventLoopType: type, comptime
                     return;
 
                 if (comptime reload_immediately) {
-                    bun.reloadProcess(bun.default_allocator, Output.enable_ansi_colors);
+                    bun.reloadProcess(bun.default_allocator, true);
```

Comment:
The problem is what if the terminal doesn’t support ansi colors? Instead of clearing the screen, it will write some strange-looking characters. 

I think the issue is that Output.enable_ansi_colors is returning false for something it shouldn’t be returning false. What’s the case where the terminal isn’t reloading with —watch?

Reply:
`bun --watch test`, see #3951